<launch>

    <!-- Image Rectifier -->
    <node pkg="interactive_yolo_image" exec="rectifier_node" name="rectifier_node" output="screen">
        <remap from="interactive_yolo/rect_camera_compressed" to="/camera_2d_wide_full_hd/image/compressed"/>
        <remap from="interactive_yolo/rect_camera_info" to="camera_2d_wide_full_hd/camera_info"/>
    </node>
    <!-- Database -->
    <node pkg="interactive_yolo_database" exec="database_node" name="database"/>
    <!-- Model -->
    <node pkg="interactive_yolo_model" exec="model_node" name="model" output="screen">
        <remap from='interactive_yolo/model_input_image' to='interactive_yolo/image_rect'/>
    </node>

</launch>